sdk/src/interfaces.ts

```diff
@@ -1,20 +1,30 @@
-import { VM } from './VM';
-
 export interface RequestData {
-  type: string,
+  type: string;
   payload: {
     __reqid?: string;
     [key: string]: any;
-  }
+  };
 }
 
 export interface Project {
-  files: {[path: string]: string};
+  files: {
+    [path: string]: string;
+  };
   title: string;
   description: string;
-  template: string;
+  template:
```

Comment:
IMO we should leave this as just a `string` type, in case we introduce new template types in the future and users are on an older version of the SDK.

Reply:
Reverted back to `string`.